e2e/tests/manageCases/caseProgression/completeOrderServiceOfApplication/completeOrderServiceOfApplication.spec.ts

```diff
@@ -3,7 +3,7 @@ import Config from "../../../../config";
 import createDaCitizenCourtNavCase from "../../../../common/createCaseHelper";
 import { Helpers } from "../../../../common/helpers";
 import config from "../../../../config";
-import { CompleteTheOrder } from "../../../../journeys/manageCases/caseProgression/completeTheOrder/completeTheOrder";
+import { CompleteOrderServiceOfApplication } from "../../../../journeys/manageCases/caseProgression/completeOrderServiceOfApplication/completeOrderServiceOfApplication.ts";
 
 test.use({ storageState: Config.sessionStoragePath + "caseWorker.json" });
 
```

Comment:
probably need to amend test titles/descriptions to reflect completing service of application WA